__tests__/installer.test.ts

```diff
@@ -249,6 +249,21 @@ describe('setup-node', () => {
 
     let expPath = path.join(toolPath, 'bin');
 
+    expect(getExecOutputSpy).toHaveBeenCalledWith(
```

Comment:
@dmitry-shibanov don't you think such check is better?

Reply:
Thank you